.github/pull_request_template.md

```diff
@@ -0,0 +1,59 @@
+# Summary | Résumé
+
+> 1-3 sentence description of the changed you're proposing.
+> Please include the GitHub issue number or relevant acceptance criteria.
+> For any UI changes please include screenshots at different breakpoints.
+
+---
+
+> Description en 1 à 3 phrases de la modification proposée, avec un lien vers le
+> problème (« issue ») GitHub ou la fiche Trello, le cas échéant.
+
+# Test instructions | Instructions pour tester la modification
+
+> Sequential steps (1., 2., 3., ...) that describe how to test this change. This
```

Comment:
This can be really open to interpretation... maybe we should set what level of granularity we're looking for?